topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.md

```diff
@@ -0,0 +1,355 @@
+>
+> > ### {% icon solution %} Solution
+> >
+> > The main peak, around 1700 bp, corresponds approximately to the length of the gene coding for 16S rRNA. There is also a secondary peak around 200 bp, which may be due to __truncated amplifications__, or as a result of __non-specific hybridization of primers__ used for PCR.
+> >
+> {: .solution}
+{: .question}
+
+
+![FastQC plots quality](../../images/metagenomics-nanopore/fastqc_per_base_sequence_quality_plot.png "Per base sequence quality")
+
+
+
+If we examine figure 3, we can see that up to 3000 bp the quality of our sequencing data is around a __Phred score__ of 12, which is a relatively low value compared to other sequencing technologies. This is explained because Nanopore reads poses __high error rates in the basecalled reads__ (10% as compared to 1% for Illumina). However, Nanopore sequencing generates __very long reads__ (in theory only limited by the mechanisms of extraction of the genetic material), enabling the sequencing of the complete 16S rRNA gene, which makes it possible to identify bacterial taxa at higher resolution.
+
+![FasTQC plots CG](../../images/metagenomics-nanopore/fastqc_per_sequence_gc_content_plot.png "Per sequence GC content")
+
+Let's have a look at figure 4, which shows the content of the GC by sequence.
+
+>
+> ### {% icon question %} Questions
+>
+> What can cause GC content to show a bimodal peak?
+>
+> > ### {% icon solution %} Solution
+> >
+> > There are several possible causes for the presence of twin peaks in the GC content. One possible explanation is the presence of adapters in the sequence. Another possible cause is some kind of contamination, such as chimeras.
+> >
+> {: .solution}
+>
+{: .question}
+
+> ### {% icon comment %} Comments
+> For more information on the topic of quality control, please see our training materials [here](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial.html).
+{: .comment}
+
+
+# Improve the dataset quality
+
+In order to improve the quality of our data, we will use two tools presented above, __porechop__ and __fastp__.
+
+## Adapter and chimera removal with porechop
+
+Nanopore sequencing technology requires to ligate adapters to both ends of genomic material to facilitate the strand capture and loading of a processive enzyme at the 5'end, boosting the effectiveness of the sequencing process. Adapter sequences should be removed because they __can interfere with aligment of reads__ to 16S rRNA gene reference database, for which we will use the __porechop__ tool ({% cite rrwick2017 %}).
+
+> ### {% icon comment %} Comment
+> If you are interested in Nanopore sequecing technology, you can find more information in {% cite Jain2016 %}.
+> 
+{: .comment}
+
+
+On the other hand, __chimeric sequences__ are considered a contaminant and should be removed because they can result in artificial inflation of the microbial diversity. With __Porechop__ you can eliminate them. 
```

Comment:
```suggestion
On the other hand, __chimeric sequences__ are considered a contaminant and should be removed because they can result in artificial inflation of the microbial diversity. With {% tool [Porechop](toolshed.g2.bx.psu.edu/repos/iuc/porechop/porechop/0.2.3) %} you can eliminate them. 
```